python/cugraph/cugraph/dask/community/triangle_count.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from dask.distributed import wait, default_client
+from cugraph.dask.common.input_utils import get_distributed_data
+
+import cugraph.dask.comms.comms as Comms
+import dask_cudf
+import cudf
+
+from pylibcugraph.experimental import (
+    ResourceHandle,
+    GraphProperties,
+    MGGraph,
+    triangle_count as pylicugraph_triangle_count
+    )
```

Comment:
`ResourceHandle`, `GraphProperties`, and `MGGraph` have all been promoted so we shouldn't be importing them from experimental, otherwise users will get the "experimental-promoted" warning.

Also, can you promote `triangle_count` in pylibcugraph to stable in this PR too?  I think it's unlikely to go away/move/change in a way that would break users so I think we can safely promote it now.

Reply:
Done